src/python/pants/backend/plugin_development/tools/check_names.py

```diff
@@ -0,0 +1,207 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
copyright is now 2022 hehe

Reply:
😜

Reply:
On that note, I notice that the year is never bumped for existing files.. (so from that perspective, 2021 is correct, as the file was created then), but I wonder if the year shouldn't be updated, or perhaps add the current year, so it reads:
```
# Copyright 2014 - 2022 Pants project contributors (see CONTRIBUTORS.md).
```

Reply:
No need to update old files: https://hynek.me/til/copyright-years/

I think the idea of updating to 2022 is because it will be _merged_ in 2022. It joins pantsbuild/pants intellectual property only when merged iiuc? Altho don't quote me, definitely not an IP lawyer hah.

Reply:
haha, ok. me either.

Reply:
In that blog post, he also mentions dropping the year altogether. Could be an idea!? :)

Reply:
(it's not like it's a secret what year it was introduced, given the git history..)

Reply:
I'm on board with that :) The year doesn't seem helpful.

Reply:
The year should be the first year the file came into existence and therefore the earliest year for which copyright might be asserted.

Reply:
Personally I could never see the point of copyright headers at all, since the copyright holder has that right whether or not you assert it in a header. But I have discussed this with a lawyer and apparently this form is necessary, for reasons I didn't fully understand (possibly including just that it's customary and we don't want people reading anything into us not doing it)